### PR TITLE
[CORE] gemini tests for new wizard

### PR DESCRIPTION
**Pull request by mathisscott**
• another piece of #769

Signed-off-by: Scott Mathis <smathis@vmware.com>

[CORE] whoops

Signed-off-by: Scott Mathis <smathis@vmware.com>

[CORE] updating gemini screens

Signed-off-by: Scott Mathis <smathis@vmware.com>

[CORE] changing gemini test suites for animations

Signed-off-by: Scott Mathis <smathis@vmware.com>

[CORE] update gemini screens

Signed-off-by: Scott Mathis <smathis@vmware.com>

[CORE] gemini test fix

Signed-off-by: Scott Mathis <smathis@vmware.com>

[CORE] update gemini screen

Signed-off-by: Scott Mathis <smathis@vmware.com>

### CODE REVIEW

**Comment by jeeyun** [comment]
@mathisscott you might want to make the whole-page screenshots more specific (to only include the wizard part). The reason for this is that when the background changes (for example, we add a new component so the left sidenav content changes) then the gemini tests will fail. But if you think it's tolerable to update screenshots for those cases, I'm okay with leaving them as is.

**Comment by mathisscott** [comment]
I wanted one or two whole page shots so that we could tell if issues like the Firefox issue popped up. If it becomes a PITA, it's easy enough to change and I will be happy to do so.

File: gemini/tests/set2/wizard-deprecated.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+var WAIT_TIME = 5000;
+var WAIT_LOAD_TIME = 2000;
```
Comment (hippee-lee) [inline gemini/tests/set2/wizard-deprecated.js]: I looks like this is only using WAIT_TIME but not WAIT_LOAD_TIME.

Comment (mathisscott) [inline gemini/tests/set2/wizard-deprecated.js]: yup. this was a pre-existing issue. i will fix it. thanks for the heads up

Comment (mathisscott) [inline gemini/tests/set2/wizard-deprecated.js]: Done

**Pull request merged**